packages/app/src/components/SearchBar/style.css

```diff
@@ -12,7 +12,11 @@
 }
 
 .search-header-container {
-  color: #128b3e;
+  color: var(--brand);
   display: flex;
   align-items: center;
 }
+
+.search-form {
+  margin-bottom: 0.5rem !important;
```

Comment:
Why the `!important`?

Reply:
Fixed! Thanks for reminding me about this!